.github/workflows/deploy-env.yml

```diff
@@ -11,17 +11,43 @@ on:
         description: "The Elastic Beanstalk version label"
         required: true
         type: string
+      notify_honeybadger:
+        description: "Whether or not to notify HoneyBadger"
+        required: false
+        type: boolean
+      notify_new_relic:
+        description: "Whether or not to notify New Relic"
+        required: false
+        type: boolean
     secrets:
       EB_AWS_ACCESS_KEY_ID:
+        description: "The AWS Access Key ID you would like to deploy with (e.g. prod or staging)"
         required: true
       EB_AWS_SECRET_ACCESS_KEY:
+        description: "The AWS Secret Access Key you would like to deploy with (e.g. prod or staging)"
         required: true
       DEPLOY_FAILURES_SLACK_WEBHOOK_URL:
+        description: "The channel to message in on failures"
         required: true
       DEPLOY_SUCCESS_SLACK_WEBHOOK_URL:
+        description: "The channel to message in on successes"
         required: true
       LOOKUP_USER_EMAIL_SLACK_TOKEN:
+        description: "A token to look up users by email - not guaranteed to work for Orgs"
```

Comment:
This only works if you have your GItHub email matches what is in Slack _and_ your settings do not enable `Keep my email addresses private`.

Reply:
It does fail gracefully though.